test/mml_builder.js

```diff
@@ -1010,6 +1010,55 @@ suite('mml_builder use_workers=' + useWorkers, function() {
         });
     });
 
+    test('support per map markers_symbolizer_caches', function(done) {
+        var mml_store = new grainstore.MMLStore({ use_workers: useWorkers });
+
+        var mml_builder = mml_store.mml_builder({
+            dbname: 'my_database',
+            sql: SAMPLE_SQL,
+            style: DEFAULT_POINT_STYLE,
+            markers_symbolizer_caches: {
+                disabled: true
+            }
+        });
+        mml_builder.toXML((err, xml) => {
+            if (err)  { return done(err); }
+            var xmlDoc = libxmljs.parseXmlString(xml);
+            var xpath = "/Map/Parameters/Parameter[@name='markers_symbolizer_caches_disabled']";
+            var markers_symbolizer_caches_disabled = xmlDoc.get(xpath);
+            assert.equal(markers_symbolizer_caches_disabled.text(), "true");
+        });
+
+        var mml_builder = mml_store.mml_builder({
+            dbname: 'my_database',
+            sql: SAMPLE_SQL,
+            style: DEFAULT_POINT_STYLE,
+            markers_symbolizer_caches: {
+                disabled: false
+            }
+        });
+        mml_builder.toXML((err, xml) => {
+            if (err)  { return done(err); }
+            var xmlDoc = libxmljs.parseXmlString(xml);
+            var xpath = "/Map/Parameters/Parameter[@name='markers_symbolizer_caches_disabled']";
+            var markers_symbolizer_caches_disabled = xmlDoc.get(xpath);
+            assert.equal(markers_symbolizer_caches_disabled, undefined);
```

Comment:
FAQ:

Q: Why it is `undefined` when `disabled` is set to `false`?
A: Because at [some point](https://github.com/mapbox/carto/blob/516289b8d13f4a909ff48c2916a7f07143e1c90a/lib/carto/renderer.js#L161) the parameters evaluated to false are discarded and not present in the XML.

Q: Why didn't you change that behavior?
A: Because that saves some XML writing in grainstore and then some XML parsing in mapnik. Changing it can have at best undesired effects in performance and at worse undesired effects in default values at mapnik level. That turned out to be the main reason to name the var `disabled` with a `false` default value, rather than naming it `enabled` with a default value of `true`.

Reply:
For the record, the suggested change has been implemented, and the comments actually refer to [line 1045](https://github.com/CartoDB/grainstore/pull/145/files#diff-a552f335e9c7a2f1f51271a1226c774eR1045)